src/System.CommandLine.DragonFruit/targets/System.CommandLine.DragonFruit.targets

```
@@ -24,25 +24,31 @@
           Inputs="$(MSBuildAllProjects)"
           Outputs="$(_GeneratedEntryPointFile)">
 
-    <PropertyGroup>
-      <_GeneratedProgramFileContent>
-        <![CDATA[
-// <auto-generated>This file was created automatically</auto-generated>
-using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
-
-[CompilerGenerated]
-internal class AutoGeneratedProgram
-{
-  public static async Task<int> Main(string[] args)
-  {
-    return await System.CommandLine.DragonFruit.CommandLine.ExecuteAssemblyAsync(
-      entryAssembly: typeof(global::AutoGeneratedProgram).Assembly,
-      args: args);
-  }
-}
-        ]]>
-      </_GeneratedProgramFileContent>
+      <PropertyGroup>
+         <_GeneratedProgramFileContent >
+
+<![CDATA[
+        // <auto-generated>This file was created automatically</auto-generated>
+        using System.CommandLine.DragonFruit;
+        using System.Runtime.CompilerServices;
+        using System.Threading.Tasks;
+
+        [CompilerGenerated]
+        internal class AutoGeneratedProgram
+        {
+          public static async Task<int> Main(string[] args)
+          {
+            return await CommandLine.ExecuteAssemblyAsync(
```

This should be changed to fully qualify the method invocation. See #478 
```suggestion
            return await System.CommandLine.DragonFruit.CommandLine.ExecuteAssemblyAsync(
```